src/EnergyPlus/SolarCollectors.cc

```diff
@@ -1828,19 +1828,19 @@ namespace SolarCollectors {
             for (int CoverNum = 1; CoverNum <= NumCovers; ++CoverNum) {
                 if (CoverNum == 1) {
                     // calc linearized radiation coefficient
-                    tempnom = DataGlobalConstants::StefanBoltzmann *
-                              ((TempAbsPlate + DataGlobalConstants::KelvinConv) + (TempInnerCover + DataGlobalConstants::KelvinConv)) *
-                              (pow_2(TempAbsPlate + DataGlobalConstants::KelvinConv) + pow_2(TempInnerCover + DataGlobalConstants::KelvinConv));
+                    tempnom = Constant::StefanBoltzmann *
+                              ((TempAbsPlate + Constant::KelvinConv) + (TempInnerCover + Constant::KelvinConv)) *
+                              (pow_2(TempAbsPlate + Constant::KelvinConv) + pow_2(TempInnerCover + Constant::KelvinConv));
                     tempdenom = 1.0 / EmissOfAbsPlate + 1.0 / EmissOfInnerCover - 1.0;
                     hRadCoefA2C = tempnom / tempdenom;
                     // Calc convection heat transfer coefficient:
                     hConvCoefA2C = EnergyPlus::SolarCollectors::CollectorData::CalcConvCoeffBetweenPlates(
                         TempAbsPlate, TempOuterCover, AirGapDepth, this->CosTilt, this->SinTilt);
                 } else {
                     // calculate the linearized radiation coeff.
-                    tempnom = DataGlobalConstants::StefanBoltzmann *
-                              ((TempInnerCover + DataGlobalConstants::KelvinConv) + (TempOuterCover + DataGlobalConstants::KelvinConv)) *
-                              (pow_2(TempInnerCover + DataGlobalConstants::KelvinConv) + pow_2(TempOuterCover + DataGlobalConstants::KelvinConv));
+                    tempnom = Constant::StefanBoltzmann *
+                              ((TempInnerCover + Constant::KelvinConv) + (TempOuterCover + Constant::KelvinConv)) *
+                              (pow_2(TempInnerCover + Constant::KelvinConv) + pow_2(TempOuterCover + Constant::KelvinConv));
```

Comment:
So this turns out to be adding the same constant twice in a few places. Use constexpr KelvinConvTimesTwo = 2 * KelvinConv ??

(TempInnerCover + TempOuterCover + KelvinConv + KelvinConv)

Reply:
They are equally efficient. If there are multiple `constexpr`s in an expression and they can be combined, the compiler will combine them and they will be a single number in the generated code.